index.html

```diff
@@ -4,9 +4,41 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="styles.css">
 </head>
 <body>
+
+<header class="wrapper"> 
+    <h1>Melanie Cascante</h1>  
+
+     <section class="geometrytop">
+        <img src="images/image1.png">
+    </section>
+
+    <div class="toolbar">
+        <nav class="ic_menu">
+            <img src="images/ic_menu.png">
+        </nav>
+    </div>
+</header>
+
+    <section class="greeting">
+        <h3>Hello! I am a software developer! I can help you build a product, feature or website. Take a look of my works. If you like what you see and have a project you need coded, don't hesitate and contact me.</h3>
```

Comment:
As I mentioned before ☝🏽 the `hx` tag here should be the text you added above (Melanie Cascante) and then this should become a paragraph `p` tag.  🙂